examples/offloader.js

```diff
@@ -12,17 +14,17 @@ var fs = require('fs');
 
 var client = new SteamUser();
 var manager = new TradeOfferManager({
-	"steam": client, // Polling every 30 seconds is fine since we get notifications from Steam
```

Comment:
@DoctorMcKay it's not really valid JSON because JSON can't contain defined variables, and `client` is the one :D
The fact that it's a valid JSON is not the case for this file (though it is for other files in `resources` folder in particular).